analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ToStringShouldNotReturnNull.cs

```diff
@@ -0,0 +1,134 @@
+﻿public static class Condition
```

Comment:
Can you add UT with the recursive thing that failed in ITs?

Reply:
That is already there?!

``` C#
public class ReturnsNullViaNestedTenary
{
    public override string ToString() => // Noncompliant
        Condition.When()
         ? (Condition.When() ? null : "something") 
         : (Condition.When() ? "something" : null);
}
```